### DISCUSSION TITLE
Log File Size

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [zwave-js/zwave-js-ui#1710] [Q&A]
Does the zwavejs2mqtt.log file ever get reset? Mine is at 183MB and takes forever (if ever) to load in the viewer. I so wish we had terminal access to these files in HA OS. I miss being able to tail -f | grep them in OZW 1.4. Maybe there are capabilities in the dashboard that I haven't discovered yet.

**Comment by robertsLando** [comment]
ATM that files grows forever, the only fix to this is to introduce a log file rotation system, there is already a Feature request for this :) I could give it some more priority

**Reply by GrizzlyAK** [reply]
Not a big deal. I can always occasionally delete it, right? Or download it if I want to keep it for some reason, and then delete it. I had been doing some debugging too, so that may be why it was so large. You probably have more important things to focus on that this. I guess I was curious to if restarting the app reset it or if it just picked up where it left off. 

BTW, since my upgrade to zwavejs/zwavejs2mqtt, things have been running great! I had a bit of a hiccup the other day that was really weird following a HA upgrade (which you may have seen [here](https://github.com/zwave-js/node-zwave-js/discussions/3363)). Other than that, things are blazingly fast. I use WS, so I only use the Control Panel, but it is awesome. THANKS!

**Reply by robertsLando** [reply]
> I can always occasionally delete it, right

Sure but I agree that a rotation could be useful here. Glad you like it :)
